ui/timeline/src/commonMain/kotlin/social/androiddev/timeline/TootContent.kt

```diff
@@ -45,14 +49,18 @@ fun TootContent(
         if (message != null) {
             Text(
                 modifier = Modifier.fillMaxWidth(),
-                text = message,
+                text = buildAnnotatedString {
+                    append(message)
```

Comment:
were you appending multiple text at some point? Shouldn' this just be `text= message`

Reply:
we're going to have to format the html. I figured this allows us to split the message by html token, format and put back together. I'd use AndroidView and html builder if this was android only, not sure what to do on other platforms if we don't hand parse